xesmf/tests/test_frontend.py

```diff
@@ -305,6 +305,15 @@ def test_regrid_dataarray(use_cfxr):
     xr.testing.assert_identical(dr_out_4D['time'], ds_in2['time'])
     xr.testing.assert_identical(dr_out_4D['lev'], ds_in2['lev'])
 
+    # test transposed
+    dr_out_4D_t = regridder(ds_in2['data4D'].transpose(..., 'time', 'lev'))
+    xr.testing.assert_identical(dr_out_4D, dr_out_4D_t)
+
+    # test renamed dim
+    if not use_cfxr:
+        dr_out_rn = regridder(ds_in2.rename(y='why')['data'])
+        xr.testing.assert_identical(dr_out, dr_out_rn)
+
```

Comment:
is it also working from and to Locstream?

Reply:
I just added some tests, and yes it does work!